extraction/text_extractor.py

```diff
@@ -0,0 +1,41 @@
+"""This script checks whether DOM has text tag or not and
+creates and returns the Text object"""
+
+import bs4
+from extraction.data_models.text import Text
+
+TEXT_TAGS = {'p', 'span', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'code'}
+HEADING_TAGS = {'h1', 'h2', 'h3', 'h4', 'h5', 'h6'}
+
+
+def empty_text(_str):
```

Comment:
start with _ if private method

Reply:
Ideally, this should move to string_utils.py as static method

Reply:
Moved to utils.